code/modules/clothing/head/helmet.dm

```diff
@@ -345,6 +345,92 @@
 	cold_protection = HEAD
 	min_cold_protection_temperature = SPACE_SUIT_MIN_TEMP_PROTECT
 	armor = list("melee" = 10, "bullet" = 5, "laser" = 5,"energy" = 5, "bomb" = 5, "bio" = 50, "rad" = 20, "fire" = -10, "acid" = 0, "wound" = 5)
+//monkey sentience caps
+
+/obj/item/clothing/head/helmet/monkey_sentience
+	name = "monkey mind magnification helmet"
+	desc = "A fragile, circuitry embedded helmet for boosting the intelligence of a monkey to a higher level. You see several warning labels..."
+
+	icon_state = "monkeymind"
+	item_state = "monkeymind"
+	strip_delay = 100
+	var/mob/living/carbon/monkey/magnification = null ///if the helmet is on a valid target (just works like a normal helmet if not (cargo please stop))
+	var/polling = FALSE///if the helmet is currently polling for targets (special code for removal)
+
+/obj/item/clothing/head/helmet/monkey_sentience/Initialize()
+	. = ..()
+	icon_state = "[icon_state][rand(1,3)]"
+
+/obj/item/clothing/head/helmet/monkey_sentience/examine(mob/user)
+	. = ..()
+	. += "<span class='boldwarning'>---WARNING: REMOVAL OF HELMET ON SUBJECT MAY LEAD TO:---</span>"
+	. += "<span class='warning'>BLOOD RAGE</span>"
+	. += "<span class='warning'>BRAIN DEATH</span>"
+	. += "<span class='warning'>PRIMAL GENE ACTIVATION</span>"
+	. += "<span class='warning'>GENETIC MAKEUP MASS SUSCEPTIBILITY</span>"
+	. += "<span class='boldnotice'>Ask your CMO if mind magnification is right for you.</span>"
```

Comment:
```suggestion
	. += span_boldwarning("---WARNING: REMOVAL OF HELMET ON SUBJECT MAY LEAD TO:---")
	. += span_warning("BLOOD RAGE")
	. +=  span_warning("BRAIN DEATH")
	. += span_warning("PRIMAL GENE ACTIVATION")
	. +=  span_warning("GENETIC MAKEUP MASS SUSCEPTIBILITY")
	. += span_boldnotice("Ask your CMO if mind magnification is right for you.")
```

idk but you probavbly could do . += the entire thing and add <br> to the end and \ or \n\ to switch lines